src/Core/src/Platform/Windows/LabelHtmlHelper.cs

```diff
@@ -24,7 +24,7 @@ public static void ParseText(XElement? element, InlineCollection inlines, ILabel
 				return;
 
 			var currentInlines = inlines;
-			var elementName = element.Name.ToString().ToUpper();
+			var elementName = element.Name.ToString().ToUpperInvariant();
```

Comment:
This one really would break running on a Turkish locale... It switches on `internal const string ElementI = "I";`.